contracts/src/v0.8/ccip/test/pools/GHO/GhoBaseTest.t.sol

```diff
@@ -0,0 +1,307 @@
+    UpgradeableTokenPool(s.pools[fromId]).applyChainUpdates(chainUpdate);
+
+    chainUpdate[0].remoteChainSelector = uint64(fromId);
+    chainUpdate[0].remotePoolAddress = abi.encode(s.pools[fromId]);
+    chainUpdate[0].remoteTokenAddress = abi.encode(s.tokens[fromId]);
+    UpgradeableTokenPool(s.pools[toId]).applyChainUpdates(chainUpdate);
+    vm.stopPrank();
+  }
+
+  function _addBridge(UtilsStorage storage s, uint256 chainId, uint256 bucketCapacity) internal {
+    require(s.tokens[chainId] == address(0), "BRIDGE_ALREADY_EXISTS");
+
+    s.chainsList.push(chainId);
+
+    // GHO Token
+    GhoToken ghoToken = new GhoToken(AAVE_DAO);
+    s.tokens[chainId] = address(ghoToken);
+
+    // UpgradeableTokenPool
+    address bridgeTokenPool = _deployUpgradeableBurnMintTokenPool(
+      address(ghoToken),
+      RMN_PROXY,
+      ROUTER,
+      OWNER,
+      PROXY_ADMIN
+    );
+    s.pools[chainId] = bridgeTokenPool;
+
+    // Facilitator
+    s.bucketCapacities[chainId] = bucketCapacity;
+    vm.stopPrank();
+    vm.startPrank(AAVE_DAO);
+    ghoToken.grantRole(ghoToken.FACILITATOR_MANAGER_ROLE(), AAVE_DAO);
+    ghoToken.addFacilitator(bridgeTokenPool, "UpgradeableTokenPool", uint128(bucketCapacity));
+    vm.stopPrank();
+  }
+
+  function _updateBridgeLimit(UtilsStorage storage s, uint256 newBridgeLimit) internal {
+    vm.stopPrank();
+    vm.startPrank(OWNER);
+    UpgradeableLockReleaseTokenPool(s.pools[0]).setBridgeLimit(newBridgeLimit);
+    vm.stopPrank();
+  }
+
+  function _updateBucketCapacity(UtilsStorage storage s, uint256 chainId, uint256 newBucketCapacity) internal {
+    s.bucketCapacities[chainId] = newBucketCapacity;
+    vm.stopPrank();
+    vm.startPrank(AAVE_DAO);
+    GhoToken(s.tokens[chainId]).grantRole(GhoToken(s.tokens[chainId]).BUCKET_MANAGER_ROLE(), AAVE_DAO);
+    GhoToken(s.tokens[chainId]).setFacilitatorBucketCapacity(s.pools[chainId], uint128(newBucketCapacity));
+    vm.stopPrank();
+  }
+
+  function _getCapacity(UtilsStorage storage s, uint256 chain) internal view returns (uint256) {
+    require(!_isEthereumChain(chain), "No bucket on Ethereum");
+    (uint256 capacity, ) = GhoToken(s.tokens[chain]).getFacilitatorBucket(s.pools[chain]);
+    return capacity;
+  }
+
+  function _getLevel(UtilsStorage storage s, uint256 chain) internal view returns (uint256) {
+    require(!_isEthereumChain(chain), "No bucket on Ethereum");
+    (, uint256 level) = GhoToken(s.tokens[chain]).getFacilitatorBucket(s.pools[chain]);
+    return level;
+  }
+
+  function _getMaxToBridgeOut(UtilsStorage storage s, uint256 fromChain) internal view returns (uint256) {
+    if (_isEthereumChain(fromChain)) {
+      UpgradeableLockReleaseTokenPool ethTokenPool = UpgradeableLockReleaseTokenPool(s.pools[0]);
+      uint256 bridgeLimit = ethTokenPool.getBridgeLimit();
+      uint256 currentBridged = ethTokenPool.getCurrentBridgedAmount();
+      return currentBridged > bridgeLimit ? 0 : bridgeLimit - currentBridged;
```

Comment:
Could just do `>=` to be cleaner imo

Reply:
[cde01d7](https://github.com/aave/ccip/pull/19/commits/cde01d7cf76fe274bc839be160cdd86c6e13d401)